### ISSUE TITLE
Characet Concept Art

### ISSUE THREAD

**Issue by Sui-nyan** [kida31/untitled-plant-game#41]

**Comment by felix-schindler** [comment]
@sopikopi ist das Konzept fertig mit dem was du heute gezeigt hast, oder wird noch mehr zur Umsetzung benötigt?

**Comment by lavenderaddict** [comment]
@sopikopi is the concept finished?
can we close this issue and make a "make character sprite" issue?

**Comment by sopikopi** [comment]
yes, next up is the development of the actual sprite with animations to be used in the game.
